web/html/src/components/dialog/ReactDialog.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
```

Comment:
Here and elsewhere, that it's a **React**Dialog is an implementation detail, I think it might be better to perhaps call this just Dialog and the old one LegacyDialog or something similar. In that case when someone else comes along they won't have any question marks about which one they should be using.

Reply:
Right... sounds like a better naming, though that one will touch more files. At least it will help make the legacy ones easier to find